hooks/event-conversion-utils.ts

```diff
@@ -164,35 +195,49 @@ class EventsAirtableRecords extends AirtableRecords {
       nextPage()
     })
 
-    return Promise.resolve(seminarSeriesEvents)
+    const sortedSeminarSeriesEvents = this.sortEventsByStartDate(seminarSeriesEvents, days > 0 ? 'asc' : 'desc')
+    return Promise.resolve(sortedSeminarSeriesEvents)
   }
 
-  async convertToCommunityEvent (record: any): Promise<CommunityEvent> {
+  /**
+   * Convert an Airtable record to a CommunityEvent.
+   * @param record The Airtable record to convert.
+   * @returns The converted CommunityEvent.
+   */
+  async convertToCommunityEvent (record: Record<string, any>): Promise<CommunityEvent> {
     const event = {
-      title: this.getName(record),
-      types: this.getTypes(record),
-      image: await this.getImage(record),
-      location: this.getLocation(record),
-      regions: this.getRegions(record),
+      endDate: record.get(this.recordFields!.endDate) || '',
+      location: record.get(this.recordFields!.location) || WORLD_REGIONS.tbd,
+      regions: record.get(this.recordFields!.regions) || [WORLD_REGIONS.tbd],
+      title: record.get(this.recordFields!.name) || '',
+      to: record.get(this.recordFields!.website) || '',
+
       date: this.formatDates(...this.getDates(record)),
+      image: await this.getImage(record),
       startDate: this.getStartDate(record),
-      endDate: this.getEndDate(record),
-      to: this.getWebsite(record)
+      startDateAndTime: this.formatTime(record.get(this.recordFields!.startDateAndTime) || null),
+      types: this.getEventTypes(record)
     }
     return event
   }
 
-  async convertToSeminarSeriesEvent (record: any): Promise<SeminarSeriesEvent> {
+  /**
+   * Convert an Airtable record to a SeminarSeriesEvent.
+   * @param record The Airtable record to convert.
+   * @returns The converted SeminarSeriesEvent.
+   */
+  async convertToSeminarSeriesEvent (record: Record<string, any>): Promise<SeminarSeriesEvent> {
     const event = {
+      endDate: record.get(this.recordFields!.endDate) || '',
+      institution: record.get(this.recordFields!.institution) || '',
+      location: record.get(this.recordFields!.location) || WORLD_REGIONS.tbd,
+      speaker: record.get(this.recordFields!.speaker) || '',
+      title: record.get(this.recordFields!.name) || '',
+      to: record.get(this.recordFields!.website) || '',
+
```

Comment:
same question as above... is there a reason for the empty line?

Reply:
Same as [here](https://github.com/Qiskit/qiskit.org/pull/2941/files/33ff37851d4ae9fb4ab0055e892b140ef654979f#r1098393713):

> I was just trying to visually separate the properties that get their data using `record.get()` from the one using other more complex functions. Not really necessary, just thought that it helps when reading.

Reply:
sounds good @eddybrando 👍🏼